clients/dos/src/main/kotlin/DosModel.kt

```diff
@@ -58,10 +58,11 @@ data class PackageInfo(
 @Serializable
 data class ScanResultsRequestBody(
     /**
-     * The list of purls to get scan results for. In case multiple purls are provided, it is assumed that they all
-     * refer to the same provenance (like a monorepo).
+     * The list of packages to get scan results for. In case multiple packages are provided, it is assumed that they all
+     * refer to the same provenance (like a monorepo). If only some of the packages exist in the DOS database, new purl
+     * bookmarks will be added for the missing packages (hence the need for the declared license here).
```

Comment:
Can you help me understand what "purl bookmarks" refers to?

Reply:
I'll try :sweat_smile: So in the case of monorepos and such, since we identify the package to return scan results (and package configurations) for on DOS side based on the purl, then if there is some source that has already been scanned on DOS, but we have only some of these purls saved coming in the request, we want to save the missing ones as well, so that we know to return results/configurations for them in future requests